src/Adapters/Files/FilesAdapter.js

```diff
@@ -56,6 +59,37 @@ export class FilesAdapter {
    * @return {string} Absolute URL
    */
   getFileLocation(config: Config, filename: string): string {}
+
+  /** Validate a filename for this adaptor type (optional)1G
+   *
+   * @param {string} filename
+   *
+   * @returns {null|*|Parse.Error} null if there are no errors
+   */
+  // TODO: Make this required once enough people have updated their adaptors
+  // validateFilename(filename): ?Parse.Error {}
+}
+
+/**
+ * Default filename validate pulled out of FilesRouter.  Mostly used for Mongo storage
```

Comment:
Replace with something like /** Check if filename is valid

Reply:
Done